Find a functionally equivalent code:android.view.textservice.TextInfo.getSequence:COMMENT
Method Modifier: public      
Comment:/**
 * @return the sequence of TextInfo
 */

@@ -1,3 +1,3 @@
 {
-    return mSequence;
+    return mSequenceNumber;
 }

